blockchain/fullblocktests/generate.go

```diff
@@ -906,6 +916,10 @@ func Generate(includeLargeReorg bool) (tests [][]TestInstance, err error) {
 		})
 	}
 
+	// Checking that duplicate blocks are not allowed, done
```

Comment:
What changes were made to the `HaveBlock` function?

Reply:
Changes in HaveBlock would be required when we would be integrating it into the working code.
No need in this PR

Reply:
Ok then shouldn't it be removed?

Reply:
I have added this test here from a different PR, will edit the comment.
Right now this test just checks for duplicate block, which was not being done earlier in fullblockstest